### PR TITLE
add command overrides

### PR DESCRIPTION
**Pull request by Vektah**
Often when running a one off task you want to specify the command to run.

eg for running migrations:
```bash
ecs-run-task --file taskdef.yml migrate
```

if there are multiple containers running you will need to specify which one with --service
```bash
ecs-run-task --file taskdef.yml --service app migrate
```

### CODE REVIEW

**Comment by lox** [comment]
I reckon we should keep this simple and only allow overriding the service named with `--service` and then arguments are the command. Then we can have `--service` inferred for most tasks. 

Unless you can think of a common usecase for multiple command overrides?

**Comment by Vektah** [comment]
Yeah I went that way originally but backed out because it required a change to the struct being passed in. The usage is much nicer that way. 

Tbh Ive never used an ECS service with more than one task. You can't set the number of containers to run per task which really decreases its usefulness.

**Comment by lox** [comment]
Feel free to make the struct change!

File: main.go [hunk 1]
```diff
@@ -71,7 +85,11 @@ func main() {
 		return nil
 	}
 
-	app.Run(os.Args)
+	err := app.Run(os.Args)
+	if err != nil {
+		log.Println(err.Error())
```
Comment (lox) [inline main.go]: If you are going to use the log package, I'd probably just go for `log.Fatal(err)`

File: runner/watcher.go [hunk 2]
```diff
@@ -79,7 +79,7 @@ func (lw *logWatcher) Watch(ctx context.Context) error {
 	var after int64
 	var err error
 
-	ctx, cancel := context.WithCancel(context.Background())
```
Comment (lox) [inline runner/watcher.go]: 👍

File: main.go (hunk 1 above)
Comment (Vektah) [reply main.go]: Ah fair, wouldn't want the error message getting discarded.

**Pull request merged**